tests/phpunit/test-cache.php

```diff
@@ -774,6 +774,42 @@ public function test_wp_cache_init() {
 		$this->assertEquals( $wp_object_cache->cache, $new_blank_cache_object->cache );
 	}
 
+	public function test_cache_syncronize() {
+		global $wpdb, $wp_object_cache, $table_prefix;
+
+		if ( ! $this->cache->is_lcache_available() ) {
+			$this->markTestSkipped( 'LCache is not available.' );
+		}
+
+		// Set the initial cache
+		$key = 'test_cache_syncronize';
+		$group = 'testgroup';
+		wp_cache_set( $key, 'first_val', $group );
+
+		$address = new \LCache\Address( $group, $key );
+		$apcu_key = 'lcache:' . $wp_object_cache->lcache->getPool() . ':' . $address->serialize();
+		$this->assertEquals( 'first_val', unserialize( apcu_fetch( $apcu_key )->value ) );
+
+		// L2 isn't available as a public resource, so we need to recreate
+		list( $port, $socket ) = self::get_port_socket_from_host( DB_HOST );
+		$dsn = 'mysql:host='. DB_HOST. ';port='. $port .';dbname='. DB_NAME;
+		$options = array( PDO::ATTR_TIMEOUT => 2, PDO::MYSQL_ATTR_INIT_COMMAND => 'SET sql_mode="ANSI_QUOTES"' );
+		$dbh = new PDO( $dsn, DB_USER, DB_PASSWORD, $options );
+		$dbh->setAttribute( PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION );
+		$l2 = new \LCache\DatabaseL2( $dbh );
+
+		// Mock a event that didn't make it to APCu
+		$address = new \LCache\Address( $group, $key );
+		$event_id = $l2->set( $wp_object_cache->lcache->getPool(), $address, serialize( 'second_val' ) );
+
+		// Reloading the object cache will syncronize the event.
+		$this->assertEquals( 'first_val', wp_cache_get( $key, $group ) );
+		$this->assertEquals( 'first_val', unserialize( apcu_fetch( $apcu_key )->value ) );
+		wp_cache_init();
+		$this->assertEquals( 'second_val', wp_cache_get( $key, $group ) );
```

Comment:
Can we do something like `$first_value = uniqid('first_val_')` instead of just `first_val`? I'm concerned about these strings staying in the cache backends across repeated runs in the same environment.